fabric-compute-framework/src/main/java/com/olacabs/fabric/compute/processor/ProcessorBase.java

```diff
@@ -35,9 +35,10 @@
 public abstract class ProcessorBase {
     @Getter
     private final boolean isScheduled;
+
     @Getter
     @Setter
-    private int id;
+    private String id;
```

Comment:
I'm just worried about the performance impact of setting MDC, is it synchronised across multiple threads ? I read somewhere it is local to one particular thread. I just need a confirmation though. My changes seem to be working fine, checked with the example topology.